README.md

```diff
@@ -17,6 +17,19 @@ cross-platform way. Initially, it will provide a streamlined experience for deve
 * [Getting started with web-ext](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Getting_started_with_web-ext)
 * [Command reference](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference)
 
+Here are the commands you can run. Click on each one for detailed documentation or use `--help` on the command line, such as `web-ext --help build`.
+
+* [`build`](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/web-ext_command_reference#web-ext_build)
```

Comment:
I'm wondering if it would be better to order them based on how much they are actually used (and useful) or just ordering them alphabetically.

If we opt for ordering them by "number of times you will actually use the command", I would say:
- run, build, lint, sign, docs
or
- run, lint, build, sign, docs

Reply:
Sure. The first ordering is what I would think. I'll change it to that.

Reply:
well, actually, `build` really isn't so useful. This comes to mind after more thought: run, lint, sign, build, docs